bioblend/galaxy/invocations/__init__.py

```diff
@@ -267,6 +270,37 @@ def cancel_invocation(self, invocation_id: str) -> dict[str, Any]:
         url = self._make_url(invocation_id)
         return self._delete(url=url)
 
+    def import_invocation(
+        self, history_id: str, model_store_format: str, file_path: Optional[str] = None, url: Optional[str] = None
+    ) -> dict[str, Any]:
+        """
+        Import a invocation from an archive on disk or a URL.
+
+        :type history_id
+        :param history_id: id of the history where the invocation will be imported
+
+        :type model_store_format
+        :param model_store_format: archive type that will be imported
+
+        :type file_path: str
+        :param file_path: Path to exported history archive on disk.
+
+        :type url: str
+        :param url: URL for an exported history archive
+
+        """
+        payload: dict[str, Any] = {
+            "history_id": history_id,
+            "model_store_format": model_store_format,
+        }
+        if file_path:
+            with open(file_path, "rb") as reader:
+                payload["store_content_uri"] = "base64://" + base64.b64encode(reader.read()).decode("utf-8")
```

Comment:
That's loading everything into memory, in the client and once more on the backend, and then another time passing it into the celery task 😨. I think we need to fix the Galaxy API first before we can proceed here.

Reply:
True, that would be a problem for many invocations. I have only been playing around with small datasets.

Reply:
@Smeds Would you like to split this PR into 2, with one just containing the changes that don't need the Galaxy API fix?

Reply:
I will split it up and make 2 new pull-requests, before I close this one.

Reply:
#505 first pull-request. Does not depend on an API update

Reply:
#506 second pull-request. This one needs an updated API, also modifications to how it forwards the files.

Reply:
@nsoranzo done